website_form_builder/static/src/xml/widgets.xml

```diff
@@ -13,7 +13,7 @@
         <form>
             <div class="form-group">
                 <label for="field">Choose the new field for the form</label>
-                <select id="field" class="form-control" multiple="">
+                <select id="field" class="form-control" multiple="1" size="6">
```

Comment:
```suggestion
                <select id="field" class="form-control" multiple="" size="6">
```

`multiple` is a boolean attribute, whose valid values are either an empty string (`multiple=""`) or a string with the same name as the attribute itself (`multiple="multiple"`).

Check this out: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#Boolean_Attributes

Reply:
This one is still missing.